components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1997,7 +1997,24 @@ do_build_config() {
 do_default_build_config() {
   build_line "Writing configuration"
   if [[ -d "$PLAN_CONTEXT/config" ]]; then
-    cp -r "$PLAN_CONTEXT/config" $pkg_prefix
+    if [ -z "${HAB_CONFIG_EXCLUDE:-}" ]; then
+      # HAB_CONFIG_EXCLUDE not set, use defaults
+      config_exclude_exts=("*.sw*" "*~" "*.bak")
```

Comment:
@smith's suggestion seems good. If you have more than 20 temp files for your vim in one directory you're probably doing something wrong 😁